apps/aehttp/src/ws_int_dispatch.erl

```diff
@@ -34,21 +34,29 @@ do_execute(chain, get, QueryPayload) ->
             #{<<"height">> := Height} ->
                 {aec_chain:get_key_block_by_height(Height), {height, Height}};
             #{<<"hash">> := Hash0} ->
-                {ok, Hash} = aec_base58c:safe_decode(block_hash, Hash0),
+                Hash = decode_block_hash(Hash0),
```

Comment:
If the user asks for a specific block type, this information is lost here. What should happen if it is of the other type?

Reply:
Since both block types are currently in 1 table, [this info is not lost](https://github.com/aeternity/epoch/blob/master/apps/aecore/src/aec_db.erl#L270). When there are separate tables for key/micro blocks, this part will need refactoring too.